janus/message-transport.md

```diff
@@ -6,22 +6,38 @@ Messaging is transport agnostic. Messaging does not rely on transport for guaran
 Despite the agnostic nature of the messaging protocol, the use of a transport protocol requires agreement on conventions for the use of the said protocol. These conventions will be protocol specific and their scope is only decisions needed to transport the message on the said protocol. Anything related to the meaning, security or routing of the message is expressly out of scope.
 
 # HTTP/HTTPS
-## URL
-The endpoint URL for messages should be a simple endpoint. It should not have route info (ex. DIDs) and should be consistent for all users of the endpoint service.
+## Request
+### URL
+Endpoint URLs for Janus messages should be simple. The URL should not expose any extra information about the message (e.g., the type of message being sent). It should have no route info (e.g., DIDs) and should be consistent for all users of the endpoint service. Message types and destination DIDs are encapsulated in the encrypted messages themselves, so as not to reveal information to proxy agents and routers along the path to the ultimate destination.
 
-Example:
+Example: https://agency.example.com/msg/
+
+### Method
+The method used for Janus messages should be `POST`, and agents with HTTP interfaces that receive a different message are encouraged to return a 405 Method Not Allowed.
+
+Janus does not depend on the transport protocol for confidential communications, therefore an HTTP method can leak details about the message being sent. HTTP is resource-oriented, and Janus is message-oriented. The intent of a Janus message is embedded in the message type.
+
+### Headers
+Accept: message/vnd.janus
+Content-Type: message/vnd.janus
```

Comment:
OK. How about message/janus? You suggested application/sov-msg. Other options are application/indy-msg, application/janus-msg. The reason I suggested `message\` is because it exists and seems more applicable than application. I suggest `janus` because I think the peer-to-peer protocol could be independent of Sovrin or of Indy. What do you think?

Reply:
I like the top level type 'message'. Did not know it existed but it expresses what we want just fine. Not a lot of register sub-types for 'message', so it should be easy to get accepted into the standard. The only concern I would have is sometimes when a part of a spec is ill-used the real world implementation doesn't work well. I think we found this with TLS. The overwhelming majority are 'application', I wonder if there are implementations that assume normal top-level types (application, image, video, etc). I don't think this is too important.

I like not attaching to sovrin or indy. It is a good insight that I was not thinking of. We should be hoping that there are uses for this outside of sovrin or Indy.